src/field/index.js

```diff
@@ -94,10 +91,14 @@ class Field {
         )}`;
 
         const field = this._getInitMeta(name);
-        const defaultValue =
-            typeof initValue !== 'undefined'
-                ? initValue
-                : originalProps[defaultValueName];
+        let defaultValue;
+        if (typeof initValue !== 'undefined') {
+            defaultValue = initValue;
+        } else if (originalProps[defaultValueName]) {
+            defaultValue = originalProps[defaultValueName];
+        } else {
```

Comment:
should be  `else if (this.options.parseName)`  ? 

because `setIn` will talk a lot of compute works, if the initValues is big, this will slow down the progress.